service/controller/resource/app/desired.go

```diff
@@ -302,6 +321,20 @@ func (r *Resource) newAppSpecs(ctx context.Context, cr apiv1alpha2.Cluster) ([]k
 	return specs, nil
 }
 
+func newAppOperatorAppSpec(cr apiv1alpha2.Cluster, component releaseversion.ReleaseComponent) key.AppSpec {
+	return key.AppSpec{
+		App: releaseversion.AppOperator,
+		// Override app name to include the cluster ID.
+		AppName:         fmt.Sprintf("%s-%s", releaseversion.AppOperator, key.ClusterID(&cr)),
+		Catalog:         component.Catalog,
+		Chart:           releaseversion.AppOperator,
+		InCluster:       true,
+		Namespace:       key.ClusterID(&cr),
+		UseUpgradeForce: true,
```

Comment:
We disabled `--force` from helmclient already but this might be collied in the `service` object if we revert that option to work. 
so how about make it `false`?

Reply:
Yes good point that could cause trouble. Done.